site/app/views/forum/ForumThreadView.php

```diff
@@ -310,6 +278,94 @@ function changeName(element, user, visible_username, anon){
 		return $return;
 	}
 
+	public function createPost($thread_id, $post, $function_content, $function_date, $title_html, $first, $reply_level){
```

Comment:
Good idea putting this in a function. This should probably be private as it will only be used within ForumThreadView for now. Maybe rename this function since we have one the same name in DatabaseQueries.php. This can be changed to displayPost or something similar since it is displaying the html for the post.